### PR TITLE
Update swarm/install-manual.md

### PR DESCRIPTION
**Pull request by amincheloh**
### Describe the proposed changes

Consul container  may has a problem when restart and the IP has been change.
By adding `-advertise=<ip>` option to run command can avoid this problem.
### Project version

Any version
### Related issue or PR in another project

 https://github.com/gliderlabs/docker-consul/issues/19

### CODE REVIEW

File: swarm/install-manual.md [hunk 1]
```diff
@@ -177,9 +177,13 @@ host as one of the Swarm managers.
...
 
-3. Paste the launch command into the command line:
+3. To set up a discovery backend, use the following syntax:
 
-        $ docker run -d -p 8500:8500 --name=consul progrium/consul -server -bootstrap
+        $ docker run -d -p 8500:8500 --name=consul progrium/consul -server -bootstrap -advertise=<consul0_ip>
+        
+    Replacing `<consul0_ip>` with the IP address from the previous command, for example:
+
+        $ docker run -d -p 8500:8500 --name=consul progrium/consul -server -bootstrap -advertise=172.30.0.161
```
Comment (johndmulhausen) [inline swarm/install-manual.md]: I don't think you need this example. Keep it short!

File: swarm/install-manual.md [hunk 2]
```diff
@@ -177,9 +177,13 @@ host as one of the Swarm managers.
 
 2. From the output, copy the `eth0` IP address from `inet addr`.
 
-3. Paste the launch command into the command line:
+3. To set up a discovery backend, use the following syntax:
```
Comment (johndmulhausen) [inline swarm/install-manual.md]: s/syntax/command

**Comment by johndmulhausen** [comment]
Deploy preview failed.

Built with commit ace5a7a95204d0901ec409580fcceae158bbdb89

https://app.netlify.com/sites/docker-docs/deploys/583caf6cc4d9cc6c7e169174

**Comment by johndmulhausen** [comment]
Assigning to @mstanleyjones to merge just so I don't override her judgement. This LGTM at this point, but I leave it to you. :)

**Comment by mstanleyjones** [comment]
Merged `master` so Netlify can run. Let's make sure that builds cleanly before merging.

**Comment by mstanleyjones** [comment]
Thanks for the contribution!

**Pull request merged**